packages/suite-desktop/src/components/DesktopTitlebar/index.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
i think this file should be placed in `suite-desktop/src/support` together with the other components used in `_app.tsx` (BridgeStatus, DesktopUpdater)